### ISSUE TITLE
Feature Request: tf.layers.batchNormalization should run without "{}"

### ISSUE THREAD

**Issue by hpssjellis** [tensorflow/tfjs#661]
Please make  batchNormalization work without having to use the "{}"

This runs:
model.add(tf.layers.batchNormalization({}))  

without the  "{}" it would not run. The error I get using the API edit boxes is 

"Cannot read property 'name' of undefined"

**Comment by bileschi** [comment]
https://github.com/tensorflow/tfjs-layers/pull/308

**Comment by hpssjellis** [comment]
That was fast. Closing this one.
